addons/stock/tests/__init__.py

```diff
@@ -4,3 +4,4 @@
 import test_product
 import test_warehouse
 import test_shipment
+from . import test_stock_location_search
```

Comment:
@moylop260 you could just do `import test_stock_location_search` instead of doing `from . import test_stock_location_search`?